packages/base/src/delegate/ItemNavigation.js

```diff
@@ -335,7 +336,13 @@ class ItemNavigation {
 			this._currentIndex = 0;
 		}
 
-		const currentItem = items[this._currentIndex];
+		let currentItem = items[this._currentIndex];
+
+		if (isSlot(currentItem)) {
+			currentItem = getSlottedElements(currentItem)[0];
+		} else {
+			currentItem = items[this._currentIndex];
```

Comment:
Hi Marti, it looks good, to me the "else" part (line343-345) can be skipped, as it's a repetition of line339